server/src/main/java/org/jboss/as/server/SelfContainedContainer.java

```diff
@@ -67,58 +62,31 @@ public SelfContainedContainer() {
      *
      * @param containerDefinition The container definition.
      */
-    public void start(final List<ModelNode> containerDefinition, File content) {
+    public ServiceContainer start(final List<ModelNode> containerDefinition, ContentProvider contentProvider) throws ExecutionException, InterruptedException, ModuleLoadException {
         Thread.currentThread().setContextClassLoader(Module.getCallerModule().getClassLoader());
-        try {
-            if (java.util.logging.LogManager.getLogManager().getClass().getName().equals("org.jboss.logmanager.LogManager")) {
-                // Make sure our original stdio is properly captured.
-                try {
-                    Class.forName(org.jboss.logmanager.handlers.ConsoleHandler.class.getName(), true, org.jboss.logmanager.handlers.ConsoleHandler.class.getClassLoader());
-                } catch (Throwable ignored) {
-                }
-                // Install JBoss Stdio to avoid any nasty crosstalk, after command line arguments are processed.
-                StdioContext.install();
-                final StdioContext context = StdioContext.create(
-                        new NullInputStream(),
-                        new LoggingOutputStream(org.jboss.logmanager.Logger.getLogger("stdout"), org.jboss.logmanager.Level.INFO),
-                        new LoggingOutputStream(org.jboss.logmanager.Logger.getLogger("stderr"), org.jboss.logmanager.Level.ERROR)
-                );
-                StdioContext.setStdioContextSelector(new SimpleStdioContextSelector(context));
```

Comment:
Not even to the server.log? Though if you printed to it before WildFly was launched it should work. We could potentially get into some interesting scenarios with logging too. For example if the uses logging before they launch WildFly then the logging subsystem won't work.